doc/march_packages/march_input_device.rst

```diff
@@ -124,3 +164,33 @@ How to add a gait
 **<name_activated_state>** name of the activated state
 
 **<gait_name>** name of the gait
+
+- Make sure you return the SD addresses of the image that should be drawn in the current state in the *StateMachine.cpp*:
+
+.. code::
+   int * StateMachine::getScreenImage(){
```

Comment:
I think the first few lines here are not necessary to include. Start from the case statement